### PR TITLE
Create mobile-release-process.md

### PR DESCRIPTION
**Pull request by lfbrock**

### CODE REVIEW

File: source/process/mobile-release-process.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Notes:
+
+- All cut-off dates are based on 10am (San Francisco Time) on the day stated.
+- T-minus counts are measured in "working days" (weekdays other than major holidays in the US and Canada) prior to release day.
+
+### A. (T-minus 8 working days) Code Complete and Release Candidate Cut 
+
+1. PM:
+  - Submit pull request for changelog
+  - Check the minimum version number and submit pull request to update
```
Comment (enahum) [inline source/process/mobile-release-process.md]: minimum version number of?

File: source/process/mobile-release-process.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+### A. (T-minus 8 working days) Code Complete and Release Candidate Cut 
+
+1. PM:
+  - Submit pull request for changelog
+  - Check the minimum version number and submit pull request to update 
+2. Dev:
+  - Finish review of last pull requests for the current release and merge
+3. QA:
+  - Confirm all pull requests merged into the current release have been tested
```
Comment (enahum) [inline source/process/mobile-release-process.md]: we should add here:
- create release branch for mattermost-mobile and mattermost-redux

File: source/process/mobile-release-process.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+### B. (T-minus 7 working days) Release Candidate Testing
+
+1. QA:
+  - Post release testing instructions and spreadsheet to Native Mobile Apps channel
+2. Team:
+  - Test assigned areas of the testing spreadsheet and file any bugs found in Jira 
+  - Daily triage of hotfix candidates and decide on whether and when to cut next RC or final
+3. Dev:
+  - Make pull requests for hotfixes to the release branch
+  - Review PRs made to release branch and merge changes into both the release branch and master
```
Comment (enahum) [inline source/process/mobile-release-process.md]: I would exclude master from this, then we can merge the changes in the release branch back to master

File: source/process/mobile-release-process.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+4. Build:
+  - Verify with Release Manager before cutting any new RCs (approved fixes should be merged)
+  - Push next RC to acceptance and announce in Native Mobile Apps channel
+5. PM:
+  - Check that the release candidate is available to beta testers 
+6. QA: 
+  - Test the new RC to verify fixes merged to the release branch work
+
+### C. (T-minus 0 working days) Release Build Cut and Submitted to App Store 
+
```
Comment (enahum) [inline source/process/mobile-release-process.md]: For DEV lets add: 
- tag the release
- upload the .ipa and .apk files to GH
- merge release branch back in to master
- review and update project dependencies as needed

File: source/process/mobile-release-process.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+1. Build: 
+  - Cut final build 
+2. PM:
+  - Final changelog updates to known issues and contributors section
+  - Submit final build to App Store and Google Play Store
+  - Confirm that minimum version number is clear in update notes
```
Comment (enahum) [inline source/process/mobile-release-process.md]: again minimum version of?

File: source/process/mobile-release-process.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
+# Mobile App Release Process
+
+The Mattermost Mobile Apps team works on a monthly release process, with a new version submitted to the App Store and Google Play Store on the 5th of each month. Note: App Store approval may take a few days after the 5th.
```
Comment (hmhealey) [inline source/process/mobile-release-process.md]: Might want to clarify that it's the iOS App Store in the first sentence

**Comment by lfbrock** [comment]
Do I need to make changes to table of contents anywhere to get this to show up?

**Comment by esethna** [comment]
Updated the guide doc so it appears in the sidebar

**Pull request merged**